src/current/v23.2/cluster-virtualization-overview.md

```diff
@@ -0,0 +1,124 @@
+
+### Security
+
+By default, when connecting to a cluster using a SQL client or the DB Console, if a user does not specify whether to connect to a virtual cluster or the system virtual cluster, they are connected to the virtual cluster.
+
+To grant access to the system virtual cluster, you must connect to the system virtual cluster as a user with the `admin` role, then grant the `admin` role to the SQL user. For details, refer to [Work with virtual clusters]({% link {{ page.version.version }}/work-with-virtual-clusters.md %}#grant-access-to-the-system-virtual-cluster).
+
+### Upgrades
+
+When cluster virtualization is enabled to upgrade to a new major version, you must first replace the binary on each node and restart the node, [finalize] the upgrade on the system virtual cluster to upgrade it, then finalize the upgrade on the virtual cluster to upgrade it. This allows you to roll back an upgrade of the system virtual cluster without impacting schemas or data in virtual clusters. The system virtual cluster can be at most one major version ahead of virtual clusters. For example, when v24.1 is released, a system virtual cluster on CockroachDB v24.1 can have virtual clusters on CockroachDB v23.2.
+
+For details, refer to [Work with virtual clusters]({% link {{ page.version.version }}/work-with-virtual-clusters.md %}#upgrade-a-cluster).
+
+### Disaster recovery
+
+When cluster virtualization is enabled, the default scope of [backup]({% link {{ page.version.version }}/backup.md %}) and [restore]({% link {{ page.version.version }}/restore.md %}) commands is the virtual cluster. This means that:
+
+- A backup taken from a virtual cluster contains all data for that virtual cluster, but does not contain modifications made via the system virtual cluster such as system-level cluster settings.
+- If your deployment contains system-level customizations, you can take a separate backup of the system virtual cluster to capture them.
+- A backup of a virtual cluster can be restored as a virtual cluster in any storage cluster with cluster virtualization enabled.
+
+For more details about backing up and restoring a cluster with cluster virtualization enabled, refer to [Work with virtual clusters]({% link {{ page.version.version }}/work-with-virtual-clusters.md %}#disaster-recovery).
+
+{{site.data.alerts.callout_success}}
+For details about configuring and using Physical Cluster Replication for disaster recovery, refer to [Physical Cluster Replication Overview]({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}).
+{{site.data.alerts.end}}
+
+### Observability
+
+When cluster virtualization is enabled, cluster log messages are scoped to the virtual cluster or to the system virtual cluster, and are labeled accordingly. For example, this log message relates to a virtual cluster named `demo`:
```

Comment:
Similarly, this is a lot of detail on Obs for the overview page when there is similar material on the Work with VC page. Could this be summarized even more? With the link out to the Work with page sending users where they can find the relevant info?

Reply:
Done.